### PR TITLE
feat: Astro 2741 mat theme update

### PR DESCRIPTION
**Pull request by micahjones13**
This updates the css/scss to use our tokens where possible. There are a few values that are either opacity based or not in our tokens. For the ones not in our tokens, I also checked figma and could not find them. 

Main changes: 
- Replaced the css files with the current files from our web-components
- Re-minifyed the .min css files (I don't even think they need to exist but I left them just in case)
- Updated the `my-theme` scss files with our token names and updated some values as well. 
- Imported the design token package at root

### CODE REVIEW

File: dark/my-theme.scss [hunk 1]
```diff
@@ -1,60 +1,120 @@
...
-$primary-light: #92cbff; // --primaryLight
-$primary-light-hover: #92cbff4d; // --primaryLightHover
-$global-app-header: #172635; // --globalAppHeader
-$input-background: #ffffff; // --inputBackground
-$card-background: rgb(32, 50, 70); // --cardBackgroundColor
-$secondary-text: #d4d8dd; // --secondaryText
-$primary-dark: #3a87cf; // --primaryDark
-$list-selected: #1c3f5e; // -- Given by Daniel for List/Drawer selected background
-$table-header-background: $global-app-header;
+$color-global-secondary-300: #92cbff; // --primaryLight
```
Comment (markacianfrani) [inline dark/my-theme.scss:3]: can we import these directly from the design-token package?

File: light/my-theme.scss [hunk 2]
```diff
@@ -1,80 +1,124 @@
...
+    letter-spacing: 0,
+  )
+);
 
 // Material Shape
 @use "@material/shape" with (
-    $small-component-radius: 3px,
-    $medium-component-radius: 3px,
-    $large-component-radius: 3px
+  $small-component-radius: 3px,
```
Comment (markacianfrani) [inline light/my-theme.scss:121]: we can use the `--radius-base` token here too

File: light/my-theme.scss [hunk 3]
```diff
@@ -1,80 +1,124 @@
...
-      white-space: nowrap,
-      text-overflow: ellipsis,
-      font-weight: 400,
-    ),
-    $styles-caption: (letter-spacing: 0),
-    $styles-overline: (font-size: 0.75rem, font-weight: normal, letter-spacing: 0)
-  );
+  $font-family: unquote("Roboto, sans-serif"),
+  $styles-headline1: (
+    font-size: 2.125rem,
```
Comment (markacianfrani) [inline light/my-theme.scss:52]: We can use some of the typography tokens for these: 

```--font-heading-2-font-size```, letter-spacing, weight, etc.

**Review by markacianfrani — CHANGES_REQUESTED** [review]

File: light/my-theme.scss (hunk 3 above)
Comment (micahjones13) [reply light/my-theme.scss]: Done, none of the letter-spacing tokens matched up though.

**Review by markacianfrani — APPROVED** [review]

**Pull request merged**